test/v2/client/test_client_config.go

```diff
@@ -23,7 +23,13 @@ type TestClientConfig struct {
 	// The disperser's port
 	DisperserPort int
 	// The URL(s) to point the eth client to
+	//
+	// Either this or EthRPCURLsVar must be set. If both are set, EthRPCURLs is used.
 	EthRPCURLs []string
+	// The environment variable that contains the URL(s) to point the eth client to. Use a comma-separated list.
+	//
+	// Either this or EthRPCURLs must be set. If both are set, EthRPCURLsVar is used.
```

Comment:
> If both are set, EthRPCURLsVar is used.

This contradicts the comment above, which states "If both are set, EthRPCURLs is used."

Reply:
Good catch. The first comment was correct, this one was wrong. Fixed.